packages/calypso-products/src/feature-group-plan-map.ts

```diff
@@ -424,11 +414,14 @@ export const featureGroups: Partial< FeatureGroupMap > = {
 			FEATURE_WP_UPDATES,
 		],
 	},
-	/* END Feature groups for experiment calypso_pricing_grid_fewer_features */
 };
 
-export function resolveFeatureGroupsForFeaturesGrid(): Partial< FeatureGroupMap > {
-	if ( isAssignedToSimplifiedFeaturesGridExperimentVariant( 'simplified' ) ) {
+export function resolveFeatureGroupsForFeaturesGrid( {
+	showSimplifiedFeatures,
+}: {
+	showSimplifiedFeatures?: boolean;
+} = {} ): Partial< FeatureGroupMap > {
```

Comment:
`showSimplifiedFeatures` is temporary and should be removed. Please see https://github.com/Automattic/wp-calypso/pull/95977/files#r1825353265

Reply:
According to p5uIfZ-ggB-p2#comment-24253, the flows that need it are prospective to be removed, but not definite. Thus we will need to keep them for now.